src/main/java/org/folio/inventory/dataimport/handlers/actions/UpdateItemEventHandler.java

```diff
@@ -44,8 +45,12 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(UpdateItemEventHandler.class);
 
+  public static final String ERROR_MSG_KEY = "ERROR_MSG";
+  public static final String FAILED_EVENT_KEY = "FAILED_EVENT";
   private static final String PAYLOAD_HAS_NO_DATA_MSG = "Failed to handle event payload, cause event payload context does not contain MARC_BIBLIOGRAPHIC data or ITEM to update";
+  private static final String STATUS_UPDATE_ERROR_MSG = "Could not change item status '%s' to '%s'";
   private static final String ITEM_PATH_FIELD = "item";
+  private static final String[] protectedStatusesFromUpdate = new String[]{"Aged to lost", "Awaiting delivery", "Awaiting pickup", "Checked out", "Claimed returned", "Declared lost", "Paged", "Recently returned"};
```

Comment:
Can you please capitalize the name of the constant and use `Set` instead of array, as it is more efficient for `contains` checks.

Reply:
yes agree, thank you